### DISCUSSION TITLE
Is it possible to change rtsp(h.265) to webrtc?

### DISCUSSION THREAD

**Discussion by kimud6003** [bluenviron/mediamtx#3959] [Questions]
### Question

I want to change the rtsp of h.265 to h.264 and make it webrtc with mediamtx alone without using ffmpeg in front.
![image](https://github.com/user-attachments/assets/63f87db9-4e20-41d7-a784-b91b1bbd5aba)

**Comment by aler9** [comment]
you have to use ffmpeg inside `runOnReady` as written in the README:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#remuxing-re-encoding-compression

we won't integrate H265 encoders into the server since they are protected by patents.

**Comment by aler9** [comment]
streaming H265 tracks with WebRTC has been implemented with #4003
